pkg/midstream/write.go

```diff
@@ -112,18 +112,10 @@ func WriteMidstream(opts WriteOptions) (*Midstream, error) {
 			io.WriteString(opts.ProcessImageOptions.ReportWriter, "Copying images\n")
 
 			if opts.ProcessImageOptions.IsAirgap {
-				copyResult, err := image.CopyAirgapImages(opts.ProcessImageOptions, opts.Log)
+				err := image.CopyAirgapImages(opts.ProcessImageOptions, opts.Log)
 				if err != nil {
 					return nil, errors.Wrap(err, "failed to copy airgap images")
 				}
-
-				if err := image.UpdateInstallationEmbeddedClusterArtifacts(image.UpdateInstallationEmbeddedClusterArtifactsOptions{
-					Artifacts:   copyResult.EmbeddedClusterArtifacts,
-					KotsKinds:   opts.KotsKinds,
-					UpstreamDir: opts.UpstreamDir,
-				}); err != nil {
-					return nil, errors.Wrap(err, "failed to update installation airgap artifacts")
-				}
```

Comment:
We don't need to write back to the `installation.yaml` anymore with push results since we're just using what is in the airgap metadata.